internal/starlark/handlers/common.go

```diff
@@ -0,0 +1,153 @@
+package handlers
+
+import (
+	"awans.org/aft/internal/db"
+	"fmt"
+	"github.com/chasehensel/starlight/convert"
+	"go.starlark.net/starlark"
+	"go.starlark.net/starlarkstruct"
+)
+
+type Handler struct {
+	tx db.RWTx
+}
+
+func Aft(tx db.RWTx) *starlarkstruct.Module {
```

Comment:
Either rename this to API or namespace all of this under aft.api. I'd like to be able to aft.functions.foo(..)

Reply:
Just responded. Moved it to aft.api

Reply:
sweet